_notebooks/2020-03-27-covid19-progress-projections.ipynb

```diff
@@ -0,0 +1,2122 @@
+{
```

Comment:
<p>Organize all the helper files into a folder under <code>_notebooks/</code> don't have a separate <code>_data</code> folder.&nbsp;&nbsp;This way I can easily see all the files related to this dashboard.   </p><p> </p><p>However, before you do all this work please review all of the feedback and see if this is something you want to do, as there is a fair amount of feedback here.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/github/covid19-dashboard/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Thanks, done.</p>